src/Core/src/IAbsoluteLayout.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Layouts;
+
+namespace Microsoft.Maui
+{
+	public interface IAbsoluteLayout : ILayout 
```

Comment:
Would be great to have Z index or at least RaiseChild and LowerChild like we have in XF